app/main/views/frameworks.py

```diff
@@ -66,20 +76,26 @@ def framework_services():
 @flask_featureflags.is_active_feature('GCLOUD7_OPEN')
```

Comment:
Is it easier to do `methods=['GET', 'POST']` here?  
I've generally been splitting them into different routes.